### PR TITLE
Update openai

### PR DESCRIPTION
**Pull request by ykriger-newrelic**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by MichelLosier — CHANGES_REQUESTED** [review]
The OpenAI quickstart still points to the openai dashboards and data sources, the quickstart will need to either 

a) Remove those references to point to new dashboards and data source
b) Bring back the openai data source and redefine the install, much like what was done for the install plan in this PR

File: install/openai/install.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ target:
   destination: cloud
 
 install:
-  mode: link
```
[the remark this answers is not in the document]
Comment (ykriger-newrelic) [reply install/openai/install.yaml:11]: @MichelLosier can u share an example for how it should look like? 
If im removing the install there is no link to the `dataSourceId: llm-application`

[the remark this answers is not in the document]
Comment (MichelLosier) [reply install/openai/install.yaml:11]: Install plans which are found in the `install/` are deprecated in favor of data sources which serve a similar function

Here is how you can configure similar nerdlet linking with a data source:
https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-source-install-fields

The existing openAI data source:
https://github.com/newrelic/newrelic-quickstarts/blob/main/data-sources/openai/config.yaml

which I imagine would have a similar update

**Review by MichelLosier — CHANGES_REQUESTED** [review]
As discussed prior, since the new llm-application data source is going to be favored in the nr1 experiences, lets remove these tags on the quickstart:

```
  - NR1_addData
  - NR1_sys
 ```

**Review by MichelLosier — APPROVED** [review]

**Comment by MichelLosier** [comment]
Applying DO_NO_MERGE label until we release the data source

**Pull request merged**